Mathlib/Data/Fintype/Sigma.lean

```diff
@@ -43,3 +43,11 @@ instance PSigma.instFintype : Fintype (Σ' i, κ i) := .ofEquiv _ (Equiv.psigmaE
 
 @[simp] lemma Finset.univ_sigma_univ : univ.sigma (fun _ ↦ univ) = (univ : Finset (Σ i, κ i)) := rfl
 #align finset.univ_sigma_univ Finset.univ_sigma_univ
+
+instance Infinite.sigma_of_left {β : α → Type*} [Infinite α] [∀ a, Nonempty (β a)] :
+    Infinite ((a : α) × (β a)) :=
```

Comment:
This is really confusing notation. Can't you simply use
```suggestion
    Infinite (Σ a, β a) :=
```

Reply:
If I do that, the infoview still shows the original notation, which is why I stated it that way. 

If the notation with the Sigma is less confusing, the pp should be changed to reflect that, and it's out of scope here.

Reply:
At least, don't add extraneous brackets
```suggestion
    Infinite ((a : α) × β a) :=
```